packages/open-next/src/wrappers/docker.ts

```diff
@@ -0,0 +1,27 @@
+import { createServer } from "http";
+
+import { ResponseStream } from "../adapters/http";
+import { Wrapper } from "../adapters/types/open-next";
+
+const wrapper: Wrapper = async (handler, converter) => {
```

Comment:
Very cool. We should allow for clusters, eg: https://nodejs.org/api/cluster.html
It's just a matter of moving the block below into the `else` block.
In my testing, the vertical scaling of the multi-core added about 25% more throughput than an equivalent horizontal scale.

Reply:
Good idea, i haven't looked too much into it, is there any downsides ?

Anyway it should be really easy to customize this according to your needs. 
If you want to override this and use your own, you just have to create something that respect `Wrapper` interface and import it in `open-next.config.ts`.

And if/when we add splitting, each split could have different wrapper and converter.
Your api could be on docker, your ISR and SSG on lambda and your SSR on the edge.

Reply:
Not that I'm aware of, it just makes use of all the CPUs on the machine. All requests are distributed on the same port so there's no behavioral changes.